### ISSUE TITLE
UX on the editor title size

### ISSUE THREAD

**Issue by ckdanny** [streetwriters/notesnook#4183]
### What problem are you facing?

The automatic title change based on length can be frustrating, especially when a short title becomes disproportionately large. I believe long titles size is fine (or can be made smaller), but the issue lies in the excessively large size of short titles. It would be preferable to have a fixed title size or ensure that short titles are not disproportionately large.

![2024-01-27 00 09 36](https://github.com/streetwriters/notesnook/assets/11786201/a4ec115a-5a5b-4c86-b1c5-2cefed9baa51)

### What's the solution you'd like to see in Notesnook?

No

### What alternatives have you considered?

Make the title longer even it can be very short

### Additional context

*No response*

**Comment by thecodrr** [comment]
Hi, thank you for the suggestion. Let me clarify, you'd like a smaller title regardless of whether the number of characters in the title?

**Comment by ckdanny** [comment]
Correct, I think fixing the title size can make the UI consistent; opening every note will have the UX. The current init title size is too large. IMO, there is no benefit to the auto size adjustment on the title.

**Comment by ammarahm-ed** [comment]
This has been fixed, The title now wraps into multiple lines.
